pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -339,32 +339,18 @@ func sendAttestReqProtobuf(attestReq *attest.ZAttestReq, iteration int) {
 	}
 
 	deferKey := "attest:" + zcdevUUID.String()
-	zedcloud.RemoveDeferred(zedcloudCtx, deferKey)
 
-	buf := bytes.NewBuffer(data)
-	size := int64(proto.Size(attestReq))
 	attestURL := zedcloud.URLPathString(serverNameAndPort, zedcloudCtx.V2API,
 		devUUID, "attest")
-	const bailOnHTTPErr = false
-	_, _, rtf, err := zedcloud.SendOnAllIntf(zedcloudCtx, attestURL,
-		size, buf, iteration, bailOnHTTPErr)
-	if err != nil {
-		// Hopefully next timeout will be more successful
-		switch rtf {
-		case types.SenderStatusUpgrade:
-			log.Functionf("sendAttestReqProtobuf: Controller upgrade in progress")
-		case types.SenderStatusRefused:
-			log.Functionf("sendAttestReqProtobuf: Controller returned ECONNREFUSED")
-		case types.SenderStatusCertInvalid:
-			log.Warnf("sendAttestReqProtobuf: Controller certificate invalid time")
-		case types.SenderStatusCertMiss:
-			log.Functionf("sendAttestReqProtobuf: Controller certificate miss")
-		default:
-			log.Errorf("sendAttestReqProtobuf failed: %s", err)
-		}
-		zedcloud.SetDeferred(zedcloudCtx, deferKey, buf, size, attestURL,
-			true)
-	}
+	buf := bytes.NewBuffer(data)
```

Comment:
The other code paths check for buf == nil and log.Fatal

Reply:
Added